platform/security/idp/security-idp-server/src/test/java/org/codice/ddf/security/idp/server/IdpEndpointTest.java

```diff
@@ -782,8 +781,8 @@ public void testPassiveLoginPkiUnsupportedPost()
 
     Response response = idpEndpoint.showPostLogin(samlRequest, relayState, request);
     String responseStr =
-        StringUtils.substringBetween(
-            response.getEntity().toString(), "SAMLResponse\" value=\"", "\" />");
+        XmlSearch.evaluate(
+            "/html/body/form/input[@name='SAMLResponse']/@value", response.getEntity().toString());
```

Comment:
Replace string matcher with XPath. The string matcher got fouled because I think handlebars threw in a newline where before there was none.